### ISSUE TITLE
FbConnection.Open() throws and exception when used with unit tests.

### ISSUE THREAD

**Issue by Aliotorem** [FirebirdSQL/NETProvider#959]
When using the FirebirdClient in an xUnit-project, a NullRefernceException gets thrown every time the method FbConnection.Open() is called.
The client is used in .NET Core 3.1. Lately we used .NET Framework 4.6.1 where the unit tests just worked fine.

Exception Message:
---- System.NullReferenceException : Object reference not set to an instance of an object.

Stack trace:
 FbConnectionInternal.GetSystemWebHostingPath()
    FbConnectionInternal.GetProcessName()
    FbConnectionInternal.BuildDpb(DatabaseBase db, ConnectionString options)
    FbConnectionInternal.Connect(AsyncWrappingCommonArgs async)
    FbConnection.OpenImpl(AsyncWrappingCommonArgs async)
    FbConnection.Open()

I've attached two projects. A console application, where the written code works and an xUnit-project, where it doesn't. Both in .NET Core 3.1.

[FirebirdUnittests.zip](https://github.com/FirebirdSQL/NETProvider/files/6555062/FirebirdUnittests.zip)

**Comment by lukasweber** [comment]
Yes, we are facing the same issue on our side. It seems to be related to this method which seems to check whether the current application is a web application and tries to get `ApplicationPhysicalPath`. In there the `NullReferenceException` is caused by the type `System.Web.Hosting.HostingEnvironment` being inexistent.

https://github.com/cincuranet/FirebirdSql.Data.FirebirdClient/blob/master/Provider/src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbConnectionInternal.cs#L456:
```
    private static string GetSystemWebHostingPath()
    {
#if NETSTANDARD2_0 || NET5_0
    return null;
#else
    var assembly = AppDomain.CurrentDomain.GetAssemblies().Where(x => x.GetName().Name.Equals("System.Web", StringComparison.Ordinal)).FirstOrDefault();
    if (assembly == null)
        return null;
    // showing ApplicationPhysicalPath may be wrong because of connection pooling
    // better idea?
    return (string)assembly.GetType("System.Web.Hosting.HostingEnvironment").GetProperty("ApplicationPhysicalPath").GetValue(null, null);
#endif
   }
```

By looking at the .NET documentation, it seems like `System.Web.Hosting.HostingEnvironment` is only present when .NET Framework is used. Is it possible that this check `#if NETSTANDARD2_0 || NET5_0` is unintentially not targeting .NET Core 3.1 or is there in any reason why it is like that?
